index.html

```diff
@@ -661,24 +734,6 @@
                     mxEvent.consume(evt);
                 };
 
-                // Returns a shorter label if the cell is collapsed and no
-                // label for expanded groups
-                graph.getLabel = function(cell) {
```

Comment:
My console shows:
`cell.blockInstance.instance.get is not a function`
This error is thwarting any attempt to open the properties window. [Tried on BIGSUM block]